src/read.c

```diff
@@ -1108,6 +1108,37 @@ static avifBool isAlphaURN(const char * urn)
 // ---------------------------------------------------------------------------
 // BMFF Parsing
 
+static avifBool avifParseHandlerBox(const uint8_t * raw, size_t rawLen)
+{
+    BEGIN_STREAM(s, raw, rawLen);
+
+    CHECK(avifROStreamReadAndEnforceVersion(&s, 0));
+
+    uint32_t predefined;
+    CHECK(avifROStreamReadU32(&s, &predefined)); // unsigned int(32) pre_defined = 0;
+    if (predefined != 0) {
+        return AVIF_FALSE;
+    }
+
+    uint8_t handlerType[4];
+    CHECK(avifROStreamRead(&s, handlerType, 4)); // unsigned int(32) handler_type;
+    if (memcmp(handlerType, "pict", 4) != 0) {
+        return AVIF_FALSE;
+    }
+
+    for (int i = 0; i < 3; ++i) {
+        uint32_t reserved;
+        CHECK(avifROStreamReadU32(&s, &reserved)); // const unsigned int(32)[3] reserved = 0;
+        if (reserved != 0) {
+            return AVIF_FALSE;
+        }
```

Comment:
In many systems, encoders must set reserved fields to 0 but decoders are supposed to ignore the reserved fields. If decoders require reserved fields to be 0 and some decoders cannot be upgraded, then we will never be able to use the reserved fields in the future.

Reply:
Ah, this is good information. I'm perfectly willing to ignore the reserved value here; I just saw that it was `= 0` in the spec and took that to mean that it was required to be such.

Reply:
I've removed the check.